### PR TITLE
Update podspec file

### PR DESCRIPTION
**Pull request by mbarnach**

### CODE REVIEW

**Review by futurejones — COMMENTED** [review]
@mbarnach did you want to add `s.swift_version = '5.1'` as in the `LoggerAPI.podspec`?
I am getting ```[!] Usage of the '.swift_version' file has been deprecated! Please delete the file and use the 'swift_versions' attribute within your podspec instead.``` when running `pod lib lint`.

**Comment by mbarnach** [comment]
I'm trying without (as it was set) to see if this could overcome the issue/inconsistency we have with LoggerAPI.
It is a warning for you also? Do you think it will have deeper consequences than that? I'm still trying to figure the middle ground for Cocoapods.

**Comment by futurejones** [comment]
It shouldn't make any practical difference except to silence the lint warnings. Both `.swift-version` and `s.swift_version` should function the same.
The only thing I am not sure about is deleting the `.swift-version` file because it appears to be referenced the in `.travis.yml` for the swift version. I noticed in LoggerAPI there is both `.swift-version` and `s.swift_version`

**Comment by mbarnach** [comment]
@futurejones @dannys42 I think it makes more sense to have both podspec and swift-version files on the same version. I've updated the PR.

File: SwiftJWT.podspec [hunk 1]
```diff
@@ -1,11 +1,12 @@
 Pod::Spec.new do |s|
   s.name         = "SwiftJWT"
-  s.version      = "3.5.3"
+  s.version      = "3.6.200"
   s.summary      = "An implementation of JSON Web Token using Swift."
   s.homepage     = "https://github.com/IBM-Swift/Swift-JWT"
```
Comment (dannys42) [inline SwiftJWT.podspec:5]: Can you update the URL as well?

Comment (mbarnach) [reply SwiftJWT.podspec]: Good catch!

**Review by dannys42 — APPROVED** [review]

**Pull request merged**